src/main/java/vdm/VDMGenerator.java

```diff
@@ -148,12 +149,17 @@ private static String segment(int count, String factory, String name, String...
 							log.log(Level.WARNING, "Could not generate template", ex);
 						}
 					}
-					command = "exec " + demoCfgName + "_" + (cfgCount++) + "; startrecording";
+					command = ((AbstractExec)tick).getCommand(count++);
```

Comment:
For the exec_record case, this should be ``cfgCount++``. As it will create CFG files using that name.